profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.callbacks.inc

```diff
@@ -9,51 +9,53 @@
  * Callback functions for the nexteuropa language negotiation providers.
  */
 function nexteuropa_multilingual_language_negotiation_provider_callback($languages) {
-  $language_url = FALSE;
   $current_path = request_path();
 
+  // $_GET['q'] might not be available at this time, because
```

Comment:
$_GET['q'] can also be empty for the domain name of the site. The code is correct for that case also, but the comment seems to ignore it, which is confusing.